src/components/HeroSection.jsx

```diff
@@ -27,7 +27,11 @@ const HeroSection = () => {
           fade={false}
           className="textloop lg:h-270px lg:w-500px"
         >
-          <LoopName>Ray Dalio</LoopName>
+          <LoopName>
+            Ray
+            <br />
+            Dalio
+          </LoopName>
```

Comment:
Do we need the <br />?

Reply:
Yes because the Ray Dalio name is the shortest one and doesn't go in two lines. I would prefer to leave the br than reducing the container size because some of the other names are way longer and risk to be broken in 3 parts